x/reward/keeper/querier.go

```diff
@@ -29,3 +31,19 @@ func queryParams(ctx sdk.Context, _ []string, _ abci.RequestQuery, k Keeper, leg
 
 	return res, nil
 }
+
+func queryPool(ctx sdk.Context, _ []string, _ abci.RequestQuery, k Keeper, legacyQuerierCdc *codec.LegacyAmino) ([]byte, error) {
+	rewardAcc := k.GetRewardAccount(ctx)
+
+	pool := k.bankKeeper.GetAllBalances(ctx, rewardAcc.GetAddress())
```

Comment:
this, too.

Reply:
https://github.com/xpladev/xpla/pull/18/commits/2542f080f072a4ad4ed025cb7e1d4d7504cc2c94